eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -466,6 +493,11 @@ object Router {
   case object GetRoutingStateStreaming
   case object RoutingStateStreamingUpToDate
   case object GetRouterData
+  case object GetNodes
+  case object GetLocalChannels
+  case object GetChannels
+  case object GetChannelsMap
+  case object GetChannelUpdates
```

Comment:
You could have used the `replyTo` pattern for those new messages, but we may as well handle them all at once when the day comes.

Reply:
Agreed, I'd rather change the whole router at once (or at least everything we can update in the router at once).